src/plugins/bodhi.c

```diff
@@ -271,6 +270,36 @@ static void bodhi_print_errors_from_json(json_object *json)
     return;
 }
 
+/**
+ * Parses only name of rpm package
+ */
+static int parse_pkginfo_name(const char *pkg_full_name, char **name)
```

Comment:
Sure, I will write explaining comments.
But I found some mistake, RPM naming convention is:
name-version-release.architecture -> ("easy-1.2.3-4.fc18.x86_64")
this format is parsed (hawkey & our new code) but it seems we used it badly until now, because we get "nvr" just in format:
name-version-release -> ("dnf-plugins-core-1000.1.21-2.fc24" in runtests/bodhi/queries/pkgmgr_query)
so our parsed values "were shifted by one", ending with parsed name shorter or returned error.
The only question now is if change of this will not affect existing running solution somewhere in production, what do you think?